prov/efa/src/rdm/rxr_pkt_type_req.c

```diff
@@ -1111,7 +1116,7 @@ struct efa_rdm_ope *rxr_pkt_get_msgrtm_rxe(struct efa_rdm_ep *ep,
 	}
 
 	pkt_type = rxr_get_base_hdr((*pkt_entry_ptr)->wiredata)->type;
-	if (rxr_pkt_type_is_mulreq(pkt_type))
+	if (rxr_pkt_type_is_medium(pkt_type) || rxr_pkt_type_is_runt(pkt_type))
```

Comment:
is this `rxr_pkt_type_is_mulreq` deleted? IIRC it is equivalent to `rxr_pkt_type_is_medium(pkt_type) || rxr_pkt_type_is_runt(pkt_type)`

Reply:
At some point, I wanted to eliminate `rxr_pkt_type_is_mulreq`, but I did not do that.

Changed back here.

Reply:
changed back in new revision.